Glossary of Terms.md

```diff
@@ -83,18 +83,30 @@ A [**payload**](#def-payload) containing one specific [HTTP Response](http://www
 ### Resource
 An API [**resource**](http://www.w3.org/TR/di-gloss/#def-resource) specified by its *URI*. It can also refer to a [**set of resources**](#def-resource) matching one [**URI template**](#def-uri-template).
 
+<a name="def-resource-group"></a>
+### Resource Group
```

Comment:
I wanted to recognize them here as an important term, to be able to link explanation of the term from other places.